code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm

```diff
@@ -0,0 +1,195 @@
+/mob/living/simple_animal/hostile/abnormality/censored
+	name = "CENSORED"
+	desc = "What is this... It's too disgusting to even look at..."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "censored"
+	icon_living = "censored"
+	pixel_x = -16
+	base_pixel_x = -16
+	faction = list("censored")
+	speak_emote = list("screeches")
+	attack_verb_continuous = "attacks"
+	attack_verb_simple = "attack"
+	attack_sound = 'sound/abnormalities/censored/attack.ogg'
+	/* Stats */
+	threat_level = ALEPH_LEVEL
+	fear_level = ALEPH_LEVEL + 3
+	health = 4000
+	maxHealth = 4000
+	obj_damage = 600
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.6, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.4, PALE_DAMAGE = 1)
+	melee_damage_type = BLACK_DAMAGE
+	melee_damage_lower = 45
+	melee_damage_upper = 60
+	speed = 4
+	move_to_delay = 5
+	/* Works */
+	start_qliphoth = 2
+	can_breach = TRUE
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(80, 70, 60, 50, 40),
+						ABNORMALITY_WORK_INSIGHT = list(90, 80, 70, 60, 50),
+						ABNORMALITY_WORK_ATTACHMENT = list(70, 60, 50, 40, 30),
+						ABNORMALITY_WORK_REPRESSION = 0,
+						"Sacrifice" = 999,
+						)
+	work_damage_amount = 14
+	work_damage_type = BLACK_DAMAGE
+
```

Comment:
Genuinely it needs EGO, at least the weapon, to be fun to deal with at all. No weapon means there's literally no reward for this havoc-wreaking bastard to be picked. You know of the Fear changes that are in the works as well so I don't need to go into detail about that.